web-client/src/client_configuration.rs

```diff
@@ -0,0 +1,129 @@
+use std::str::FromStr;
+
+use tsify::Tsify;
+use wasm_bindgen::prelude::{wasm_bindgen, JsError, JsValue};
+
+use nimiq_primitives::networks::NetworkId;
+
+/// Use this to provide initialization-time configuration to the Client.
+/// This is a simplified version of the configuration that is used for regular nodes,
+/// since not all configuration knobs are available when running inside a browser.
+#[wasm_bindgen]
+pub struct ClientConfiguration {
```

Comment:
Members are missing `rustdoc`

Reply:
The members' names are self-explanatory I would say.